components/docs/tableOfContents.tsx

```diff
@@ -51,14 +51,14 @@ export function TableOfContents() {
         <Card.Body>
           <small className="docsTableOfContents">
             {navItems.map((section, idx) => {
+              const isActiveSection = router.asPath.includes(section.path);
```

Comment:
I like cleaning stuff like this up! We're using very similar logic above in the NavListItem component. Maybe we can share the logic?